osidb/constants.py

```diff
@@ -61,12 +61,12 @@
 AFFECTEDNESS_VALID_RESOLUTIONS = {
     Affect.AffectAffectedness.NEW: [
         Affect.AffectResolution.NOVALUE,
-        Affect.AffectResolution.WONTFIX,
+        Affect.AffectResolution.DEFER,
         Affect.AffectResolution.OOSS,
     ],
     Affect.AffectAffectedness.AFFECTED: [
         Affect.AffectResolution.DELEGATED,
-        Affect.AffectResolution.WONTFIX,
```

Comment:
I am not aware of that we should stop using WONTFIX. Is there any reference about it?

Reply:
It was requested and documented OSIDB-3829, but this might be better to tackle in a later PR

Reply:
typo: OSIDB-3289

Reply:
Then it is OK